pallets/pools/src/lib.rs

```diff
@@ -55,6 +55,17 @@ where
 	pub min_epoch_time: Moment,
 	pub challenge_time: Moment,
 	pub max_nav_age: Moment,
+
+	pub status: PoolStatus,
+}
+
+#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo)]
+pub enum PoolStatus {
+	Open,
+	InSubmissionPeriod,
+	/// Repayments possible, else not possible
```

Comment:
What does this mean?

Reply:
I am not sure what else is not possible. Need to check this back with @offerijns.

Reply:
I'm not sure if this comment is referring to the `InSubmissionPeriod` or the `Frozen` state?

In case of `InSubmissionPeriod`, here just originations should be blocked, nothing else.

In case of `Frozen`, I'm assuming this is the state when 1 tranche was wiped out? If so, the spec I discussed in the past with Dennis was:
```
Once a tranche is wiped out: originations should be disallowed, repayments should be allowed, redemptions from the non wiped out tranches should be allowed, investments shouldn’t be allowed in any tranche.
```

We don't need to implement this now though.

Reply:
I think besides the "per-tranche-base" actions the PR does it implement it that way.